frontend/src/utils/shares.ts

```diff
@@ -113,4 +113,40 @@ const getOddsForOutcome = (
   return oddsForOutcome
 }
 
-export { Shares, getShares, getPotentialWinnings, getOddsForOutcome }
+const getSharesPurchased = (
+  market: Market,
+  selectedOutcome: OutcomeId,
+  buyAmount: Coins,
+): Shares => {
+  const selectedPool =
+    market.possibleOutcomes.find((outcome) => outcome.id === selectedOutcome)
+      ?.poolShares.units ?? BigNumber(0)
```

Comment:
NIT: I'd probably throw an exception if we can't find this outcome instead of returning 0.